operator/controllers/internal/csiscaleoperator/csiscaleoperator_package.go

```diff
@@ -369,7 +369,7 @@ func (c *CSIScaleOperator) GenerateSnapshotterClusterRole() *rbacv1.ClusterRole
 			{
 				APIGroups: []string{snapshotStorageApiGroup},
 				Resources: []string{volumeSnapshotContentsStatusResource},
-				Verbs:     []string{verbUpdate, verbPatch},
```

Comment:
@amdabhad @ghanshyam-11 are we sure that we are not breaking anything while remove verbUpdate operation ?

Reply:
Thanks @Jainbrt ! @ghanshyam-11 even though our regression worked fine with these clusterroles and update verb had to be removed from resizer, looking at [community snapshotter clusterroles](https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml), it seems we need update verb here?

Reply:
update verb is added back by https://github.com/IBM/ibm-spectrum-scale-csi/pull/811, community snapshotter issue for reference: https://github.com/kubernetes-csi/external-snapshotter/pull/707